### PR TITLE
groestlcoin: 24.0.1 -> 25.0

### PR DESCRIPTION
**Pull request by gruve-p**
Latest stable release: https://github.com/Groestlcoin/groestlcoin/blob/master/doc/release-notes/release-notes-25.0.0.md

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by gruve-p** [comment]
Ping @SuperSandro2000

**Review by SuperSandro2000 — COMMENTED** [review]
Can you mark it broken for darwin?

**Comment by gruve-p** [comment]
> Can you mark it broken for darwin?

Force pushed a fix for Darwin.

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -35971,6 +35971,7 @@ with pkgs;
   go-exploitdb = callPackage ../tools/security/go-exploitdb { };
 
   groestlcoin  = libsForQt5.callPackage ../applications/blockchains/groestlcoin {
+    stdenv = if stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else stdenv;
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:35974]:
```suggestion
    stdenv = darwin.apple_sdk_11_0.stdenv;
```
for linux that variable is equal to stdenv.

**Comment by gruve-p** [comment]
@SuperSandro2000 Do you want me to force push or is it ok like this?

**Comment by SuperSandro2000** [comment]
force push but I just squash merged it

**Pull request merged**